python/samples/getting_started/agents/nvidia/nvidia_nim_chat_client.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import Any
+from collections.abc import Sequence
+
+from agent_framework.openai import OpenAIChatClient
+from agent_framework._types import (
+    ChatMessage, 
+    Contents, 
+    TextContent,
+    Role,
+    FunctionCallContent,
+    FunctionResultContent,
+    FunctionApprovalRequestContent,
+    FunctionApprovalResponseContent,
+    prepare_function_call_results,
+)
+
+
+class NVIDIANIMChatClient(OpenAIChatClient):
```

Comment:
Directed to @eavanvalkenburg: we have other clients based on the OpenAIChatClient, why wouldn't we want to bring this NVIDIANIMChatClient into its own package, instead of it being purely in the samples?

Reply:
Good question @moonbox3 I've asked Shawn for his opinion